programs/summerofcode/2024.md

```diff
@@ -31,6 +31,23 @@ You can find the project ideas from previous year [here](./2023.md).
 
 ### Proposals
 
+#### CNCF GOSST
+
+##### CNCF and Google Open Source Security Team GSoC Collaboration - Enhancing Security Across CNCF Ecosystem
+
+- Description: This project is a collaborative effort between the CNCF and Google's Open Source Security Team to improve security practices across various CNCF projects. The focus is on identifying and addressing security vulnerabilities, integrating security tools like OSS-Fuzz, and enhancing build and release security processes.
+- Expected Outcome:
+  * Integration or enhancement of fuzzing with [OSS-Fuzz](https://google.github.io/oss-fuzz/) for CNCF projects
+  * Remediation of known vulnerabilities within the CNCF ecosystem
+  * Improved build/release security by automating builds, releases, added build provenance, signing and improved reproducibility
+  * Enhanced [OpenSSF Scorecard](https://securityscorecards.dev/) and [CLOMonitor](https://clomonitor.io/search?foundation=cncf&page=1) scores for CNCF projects.
+- Recommended Skills: Security analysis, CI/CD practices, programming (preferably Go), knowledge of CNCF projects.
+- Expected project size: medium (~175 hour projects) or large (~350 hour projects)
```

Comment:
I think this could be a large project, but am curious what your thoughts are Dustin.

```suggestion
- Expected project size: large (~350 hour projects)
```

Reply:
Following up on this, if we focus on getting all CNCF projects to use scorecards, we can run this as a small ~90 hour project.

```suggestion
- Expected project size: small (~90 hour projects)
```